CONTRIBUTING.md

```diff
@@ -1,75 +1,76 @@
+  should be made. It may feel like a formality in some cases, but it often helps avoid wasted time, both on the part of
+  reviewers and contributors.
+
+## Issues
+
+- Issues should be used to open a discussion about something that you want to see changed.
+- Please tag individual maintainers if you would like thier input. If you don't know which maintainer to tag, just tag
+  @UMAprotocol/eng.
+- To make a change, add a feature, or fix a bug, one should start by opening an issue briefly describing what should be
+  changed and why.
+- Maintainers will generally assign the issue to a particular person to avoid multiple people working on the same issue
+  at the same time.
+- Once an issue is open and one or more maintainers have expressed corroborating opinions, the assigned person should
+  feel comfortable beginning work on the issue.
+- If an issue is a major project, consider working the maintainers to break it down into sub-issues and creating a
+  milestone to track progress.
+
+## PRs
+
+- No PRs should be opened that are not associated with one or more issues.
+- If you are not sure who to request as reviewers on your PR, request a review from @UMAprotocol/eng.
+- If you have one maintainer in mind, but do not know who else to add, you can always add @UMAprotocol/eng to
+  auto-select a second.
+- If you make updates and would like another round of reviews, use the cycle icon next to the reviewers names to
+  request a follow-up review.
+- PRs should be narrow and focused. Making multiple, unrelated changes in the same PR makes things difficult on reviewers
+  and can often slow PR progress. If this is the case, you may be asked to split the PR up.
+- PRs must get approval from _at least_ one maintainer. However, it is considered best practice to make sure all
+  comments are completely resolved before merging.
+- PRs can only be merged by maintainers. If a maintainer opened the PR, it's preferred (although, not required) to
+  allow them to merge it, themselves.
+- PR titles should be in [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format. This helps
+  inform versioning and changelogs. Please include a breaking change footer in your PR description if the change will
+  break consumers of any package. If you don't know how to correctly title your PR, this can be addressed in the review
+  process.
+
+Note on conventional commit titles: the [angular](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type) types
+are a good guideline to follow:
+
+- build: Changes that affect the build system or external dependencies
+- ci: Changes to our CI configuration files and scripts
+- docs: Documentation only changes
+- feat: A new feature
+- fix: A bug fix
+- perf: A code change that improves performance
+- refactor: A code change that neither fixes a bug nor adds a feature
+- style: Changes that do not affect the meaning of the code
+- test: Adding missing tests or correcting existing tests
```

Comment:
i have some thoughts around the "titles", the main ones being `fix` and `feature`. There are several other mentioned in the conventional commit docs: `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, test:, ie any changes which don't require a version bump. All are valid, so use at your discretion, but in practice for me ive gravitated towards: `fix`, `feature` or `improve`, where improve is an umbrella for any non versioning change.  Not necessarily advocating for that, but its an option if you want to simplify 

From the docs:

> Others: commit types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends chore:, docs:, style:, refactor:, perf:, test:, and others. We also recommend improvement for commits that improve a current implementation without adding a new feature or fixing a bug. Notice these types are not mandated by the conventional commits specification, and have no implicit effect in semantic versioning (unless they include a BREAKING CHANGE, which is NOT recommended).
A scope may be provided to a commit’s type, to provide additional contextual information and is contained within parenthesis, e.g., feat(parser): add ability to parse arrays.

Reply:
Thoughts on saying that improve is an acceptable catch-all, but the others are acceptable if you'd like to be more specific?

Reply:
that sounds good to me

Reply:
Okay, cool, added some text about that. PTAL.

Reply:
Oh, looks like you already TAL :). nevermind haha